lms/djangoapps/lms_xblock/test/test_runtime.py

```diff
@@ -231,6 +230,18 @@ def test_no_service_rendered(self):
         self.assertFalse(runtime.service(self.mock_block, 'badging'))
 
     @patch.dict(settings.FEATURES, {'ENABLE_OPENBADGES': True})
+    def test_course_badges_disabled(self):
+        self.course_id = CourseFactory.create(metadata={'issue_badges': False}).location.course_key
+        runtime = self.create_runtime()
+        self.assertFalse(runtime.service(self.mock_block, 'badging').course_badges_enabled)
+
+    @patch.dict(settings.FEATURES, {'ENABLE_OPENBADGES': True})
+    def test_course_badges_enabled(self):
```

Comment:
Nit: combining this test with the previous one and using `ddt` would be a bit cleaner.